DQM/Integration/test/runCompilationTest.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+keys=('pp_run' 'pp_run_stage1' 'cosmic_run' 'cosmic_run_stage1' 'hi_run' 'hpu_run' 'commissioning_run')
+
+# Define a function to run the python command
+run_python_command() {
+    function die { echo $1: status $2 ; exit $2; }
+    
+    entry="$1"
+    key="$2"
+
+    # Check conditions to skip certain combinations
+    if [[ "$entry" == *visualization-live_cfg.py* && ( "$key" == "pp_run_stage1" || "$key" == "cosmic_run_stage1" || "$key" == "hpu_run" ) ]]; then
+        echo "===== Skipping Test \"python3 $entry runkey=$key\" ===="
+        return
+    fi
+
+    # Otherwise, proceed with the test
+    echo "===== Test \"python3 $entry runkey=$key\" ===="
+    (python3 "$entry" runkey="$key" > /dev/null) 2>&1 || die "Failure using python3 $entry" $?
+}
+
+# Export the function for parallel execution
+export -f run_python_command
+
+# Run the keys in parallel
+parallel run_python_command ::: "${CMSSW_BASE}/src/DQM/Integration/python/clients/"*"-live_cfg.py" ::: "${keys[@]}"
```

Comment:
```
 
===== Test "TestDQMOnlineClients-compilation" ====
/data/cmsbld/jenkins/workspace/ib-run-pr-tests/CMSSW_14_1_X_2024-03-14-2300/src/DQM/Integration/test/runCompilationTest.sh: line 27: parallel: command not found
```

@smuzaffar @iarspider is there a fundamental reason why `parallel` is not supported on the test machines? (it makes execution of this test much faster).

Reply:
@mmusich , no there is no fundamental reason. No one used it that is why it is not part of cms containers. 
Ideally we like unit test to run in serial mode as scram already has started `nproc` tests in parallel. So if a test itself start doing the parallel execution then it will overload the system. Why can not add one test per configuration in BuildFile so that scram run them in parallel

Reply:
> Why not add one test per configuration in BuildFile so that scram run them in parallel?

thanks for the suggestion. I'll do it like that.